packages/contracts/contracts/optimistic-ethereum/bridge/MockCrossDomainMessenger.sol

```diff
@@ -1,70 +0,0 @@
-pragma solidity ^0.5.0;
-
-/* Interface Imports */
-import { ICrossDomainMessenger } from "./CrossDomainMessenger.interface.sol";
-
-/**
- * @title MockCrossDomainMessenger
- */
-contract MockCrossDomainMessenger is ICrossDomainMessenger {
-    /*
-     * Contract Variables
-     */
-
-    ICrossDomainMessenger targetMessenger;
-    address public crossDomainMsgSender;
-
-    /*
-     * Public Functions
-     */
-
-    /**
-     * Relays a message to a target contract.
-     * .inheritdoc ICrossDomainMessenger
-     */
-    function relayMessage(
-        address _target,
-        address _sender,
-        bytes memory _message
-    )
-        public
-    {
-        crossDomainMsgSender = _sender;
-        (bool success,) = _target.call(_message);
-        require(success, "Received message reverted during execution.");
-    }
-
-    /**
-     * Sends a message to the target messenger.
-     * .inheritdoc ICrossDomainMessenger
-     */
-    function sendMessage(
-        address _target,
-        bytes memory _message
```

Comment:
As we discussed in person, I think it makes sense to add a `gasLimit` to this function and this way we can actually keep the old mocks. As long as we assume that the "relay message" on L1 & the "replay" functionality is handled in some standardized way, then developers building for L1 / L2 shouldn't have to unit test that functionality. It should just "happen" in the background.

Reply:
Hmmm, interesting. After the Synthetix call today, I feel like having the full interface is probably useful for developers. We can always add a wrapper that ignores these extra parameters later if it seems to be useful.